### PR TITLE
AAC perftest / ITHC

### PR DESCRIPTION
**Pull request by hemantt**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by danlysiak — DISMISSED** [review]

File: k8s/namespaces/aac/manage-case-assignment/perftest.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    fluxcd.io/tag.java: glob:pr-180-*
+    hmcts.github.com/prod-automated: disabled
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/aac/manage-case-assignment:pr-180-e7c269d8
+      environment:
+        DUMMY_RESTART_VAR: true
+    nodejs:
+      image: hmctspublic.azurecr.io/aac/manage-case-assignment:pr-184-2daeecca
```
[the remark this answers is not in the document]
Comment (nathan-clark) [reply k8s/namespaces/aac/manage-case-assignment/perftest.yaml:16]: https://github.com/hmcts/cnp-flux-config/blob/f956bc8704a8895d282f6356b7990eb70247c43b/k8s/namespaces/aac/kustomization.yaml#L3

should be aac?

File: k8s/namespaces/aac/manage-case-assignment/perftest.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    fluxcd.io/tag.java: glob:pr-180-*
+    hmcts.github.com/prod-automated: disabled
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/aac/manage-case-assignment:pr-180-e7c269d8
+      environment:
+        MANAGE_CASE_S2S_AUTHORISED_SERVICES: xui_webapp,ccd_data,finrem_case_orchestration
+        MCA_DS_PROXY_URLS_ALLOWED_LIST: /searchCases.*,/internal/searchCases.*,/internal/cases.*
+        CCD_DEFINITION_STORE_API_BASE_URL: http://ccd-definition-store-api-demo.service.core-compute-demo.internal
```
Comment (jmcadam) [inline k8s/namespaces/aac/manage-case-assignment/perftest.yaml:16]: perftest instead of demo?

**Review by jmcadam — CHANGES_REQUESTED** [review]

File: k8s/namespaces/aac/manage-case-assignment/perftest.yaml (hunk 2 above)
Comment (jmcadam) [reply k8s/namespaces/aac/manage-case-assignment/perftest.yaml:16]: @hemantt one for you mate!

**Review by smathangi — APPROVED** [review]

File: k8s/perftest/common-overlay/aac/kustomization.yaml [hunk 3]
```diff
@@ -3,3 +3,6 @@ kind: Kustomization
 namespace: aac
 bases:
 - ../../../namespaces/aac
+patchesStrategicMerge:
+- ../../../namespaces/aac/manage-case-assignment/perftest.yaml
+- ../../../namespaces/aac/manage-case-assignment/ithc.yaml
```
Comment (smathangi) [inline k8s/perftest/common-overlay/aac/kustomization.yaml:8]: ithc entry should go to it's own kustomization file

**Review by smathangi — CHANGES_REQUESTED** [review]

**Review by smathangi — APPROVED** [review]

**Review by danlysiak — DISMISSED** [review]

**Review by jmcadam — APPROVED** [review]
👍

**Review by TomKimber — APPROVED** [review]

**Review by smathangi — APPROVED** [review]

**Review by hemantt — COMMENTED** [review]
done

**Pull request merged**